examples/pytorch/text-classification/run_glue.py

```diff
@@ -197,6 +197,10 @@ class ModelArguments:
             )
         },
     )
+    trust_remote_code: bool = field(
+        default=False,
+        metadata={"help": "Allow loading model with custom code"},
```

Comment:
Let's copy the original docstring

```text
                Whether or not to allow for custom models defined on the Hub in their own modeling files. This option
                should only be set to `True` for repositories you trust and in which you have read the code, as it will
                execute code present on the Hub on your local machine.
```